Included membership information (#1385)
carpentries/carpentries.org: pages/welcome-tip-sheet.md

- Community Discussions: Topics range anywhere from teaching workshops and developing curricula to building local communities and assessing the impact of our workshops globally. Community Discussions are added to our [community calendar](/community/#community-events) and are also listed in our [Etherpad](https://pad.carpentries.org/community-discussions).
- Regional Community Calls: Join a call with others from your region. [Call in details](https://pad.carpentries.org/community-discussions).

### Memberships
A Member Organisation is an organisation which has made a commitment to the growth and spread of The Carpentries. Members receive priority access to [Instructor Training](https://carpentries.github.io/instructor-training/) and [Centrally-Organised Workshops](https://carpentries.org/workshops/#workshop-organising) and guidance with capacity building at their organisation.

- [How can your organisation Become a Member?](https://carpentries.org/membership/)

> - [Benefits and tiers](https://carpentries.org/member_faq/#what-are-the-standard-membership-packages-you-offer)

Reviewer: To be consistent with capitalization, I would capitalize "Tiers"

> - [Financial Suppport](https://carpentries.org/member_faq/#is-financial-support-available)

Reviewer: Suppport should be "Support"

- [Frequently Asked Questions](https://carpentries.org/member_faq/)

> - When you are ready to begin planning a workshop with The Carpentries, please let us know 2-3 months in advance by [requesting a Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)

Reviewer: I would remove this; too much detail for tip sheet

> - You'll receive a registration code to share with your trainees. Trainees can register for any Instructor Training event that fits their schedule. Check out the [Instructor Training Calendar](https://carpentries.github.io/instructor-training/training_calendar/) for a full list of events.

Reviewer: I would remove this; too much detail for tip sheet

> - [Contact the Membership Team](emailto:membership@carpentries.org)

Reviewer: We do not add email addresses for any of the other teams on the tip sheet. If we do this, we should be consistent. Perhaps we add a "Contact Us" in the additional resources.

Author: @acrall added the following under additional resources: 

Should you have any:
general requests, please email [team@carpentries.org](mailto:team@carpentries.org)
workshop requests, please email [workshops@carpentries.org](mailto:workshops@carpentries.org)
community requests, please email [community@carpentries.org](mailto:community@carpentries.org)
membership requests, please email [membership@carpentries.org](mailto:membership@carpentries.org)

### Become an Instructor
Carpentries Instructors are the core of our community, organising and teaching workshops to spread data literacy and computational skills around the world. Becoming an Instructor is a great step to add inspiring people to your network and helps you to both develop and demonstrate your ability to teach effectively.

> - [Become an Instructor](/become-instructor/): webpage with all the details

Reviewer: I don't think this should be removed. This is an important link to have on the tip sheet

- Community Discussions: Topics range anywhere from teaching workshops and developing curricula to building local communities and assessing the impact of our workshops globally. Community Discussions are added to our [community calendar](/community/#community-events) and are also listed in our [Etherpad](https://pad.carpentries.org/community-discussions).
- Regional Community Calls: Join a call with others from your region. [Call in details](https://pad.carpentries.org/community-discussions).

### Memberships
A Member Organisation is an organisation which has made a commitment to the growth and spread of The Carpentries. Members receive priority access to [Instructor Training](https://carpentries.github.io/instructor-training/) and [Centrally-Organised Workshops](https://carpentries.org/workshops/#workshop-organising) and guidance with capacity building at their organisation.

- [Benefits and tiers](https://carpentries.org/member_faq/#what-are-the-standard-membership-packages-you-offer)

> - Additional Benefits: Contact the [Membership Team](emailto:membership@carpentries.org)

Reviewer 2: This is an option but probably not something we need to point out on the welcome tip sheet. This links to contact the team and we already have a point for this at the end.

- [Financial Suppport](https://carpentries.org/member_faq/#is-financial-support-available)
- [Frequently Asked Questions](https://carpentries.org/member_faq/)

> - [Membership Council](https://carpentries.org/member_faq/#what-is-the-carpentries-member-council)

Reviewer 2: ```suggestion
```

> - [Request an Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)

Reviewer 2: ```suggestion
- When you are ready to begin planning a workshop with The Carpentries, please let us know 2-3 months in advance by [requesting a Centrally-Organised Workshop](https://amy.carpentries.org/forms/request_workshop/)
```

> - [Workshop Checklist for Hosts](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html)

Reviewer 2: ```suggestion
- You'll receive a registration code to share with your trainees. Trainees can register for any Instructor Training event that fits their schedule. Check out the [Instructor Training Calendar](https://carpentries.github.io/instructor-training/training_calendar/) for a full list of events.
```

Reviewer 2: The workshops checklist seems too detailed for the member organisation checklist. Maybe we should take an idea back to WAT to share this once the WRF has been submitted.

> - [How can your organisation Become a Member?](https://carpentries.org/membership/): 

Reviewer 2: moved up
